### DISCUSSION TITLE
[Question] is it possible to ```import numpy``` in a embeded environment using pybind11::embed?

### DISCUSSION THREAD

**Discussion by Ma-Y** [pybind/pybind11#4544] [General]
how to get such a 3rd party module running instead of all C modules and build-in std modules?

**Comment by Dariusz1989** [comment]
If my memory serve me right... you install numpy in python that u using to build against, then u can just call it as any other module...

**Reply by Ma-Y** [reply]
thanks, i understand now.

**Comment by doomroep5** [comment]
> how to get such a 3rd party module running instead of all C modules and build-in std modules?
